sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -35,6 +35,14 @@ namespace sycl {
 inline namespace _V1 {
 namespace ext::oneapi::experimental {
 
+namespace detail {
+
+// TODO: Convert to SYCL properties
+struct is_blocked {};
+struct is_striped {};
+
+} // namespace detail
```

Comment:
I hope, it should not be the `detail` namespace, since it's public API:
```cpp
enum class group_algorithm_data_placement{
  blocked,
  striped
};
1.sycl::ext::oneapi::experimental::group_algorithm_data_placement::blocked to specify a data placement described in a).

2.sycl::ext::oneapi::experimental::group_algorithm_data_placement::striped to specify a data placement described in b).

```